randovania/gui/ui_files/main_window.ui

```diff
@@ -653,6 +651,17 @@ For more information, check [Randovania Help](help://tab_multiworld)!</string>
     <string>Verify installation</string>
    </property>
   </action>
+  <action name="menu_action_disable_sound_alerts">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Disable Sound Alerts</string>
```

Comment:
This text should show the opposite: `Enable Sound Alerts`

Reply:
Do y'all want it to be Sound Alerts to be enabled by default?
And if so, how would I do so?

Reply:
Not sure whether they should be enabled by default. On one hand, people are probably never finding the setting otherwise, on the other, I feel like more people find a sound annoying and would just turn it off. Personally leaning more towards disabled by default (due to bias)

But this was more just for phrasing to make it more sensible.

Reply:
Added to "Alert When Generation Completes" submenu, which should make the option understandable without renaming to "Enable Sound Alerts"
![382442773-09367ee5-f8bc-48c9-a4f4-fc2a756d5c20](https://github.com/user-attachments/assets/93df788a-cd92-42b3-b806-3822980837d3)